projectgenerator/gui/generate_project.py

```diff
@@ -64,6 +64,13 @@ def __init__(self, parent=None):
     def accepted(self):
         configuration = self.updated_configuration()
 
+        if not configuration.host:
+            self.txtStdout.setText(self.tr('Please set a host before creating the project.'))
+            return
+        if not configuration.database:
+            self.txtStdout.setText(self.tr('Please set a database before creating the project.'))
+            return
```

Comment:
Would be great to handle that with some warnings/colors on the dialog itself so the user doesn't need to click a button first to get the message but instead will have hints on the dialog already for recommended/required fields.